calabash/Classes/FranklyServer/LPCORSResponse.m

```diff
@@ -1,3 +1,7 @@
+#if ! __has_feature(objc_arc)
+#warning This file must be compiled with ARC. Use -fobjc-arc flag (or convert project to ARC).
```

Comment:
Migrate project to ARC so all new files are ARC by default #266

Pull requests welcome. :)